operations/configuration_management/Management_Node_Image_Customization.md

```diff
@@ -123,7 +132,7 @@ This document describes the configuration of a Kubernetes NCN image. The same st
         echo "${PARAMS}"
         ```
 
```

Comment:
I would appreciate two things from the above section [103-134]
1) Make more clear that you need to do this for EACH ncn in which you want the images to be updated. IE: Only workers.
2) Put this into a loop, so we do not have to run the same multiple commands over and over. I have included the for loop i use bellow

```
for XNAME in $(sat status | grep ncn-w00);do echo $XNAME;METAL_SERVER=$(cray bss bootparameters list --hosts "${XNAME}" --format json | jq '.[] |."params"' | awk -F 'metal.server=' '{print $2}' | awk -F ' ' '{print $1}');echo $METAL_SERVER;PARAMS=$(cray bss bootparameters list --hosts "${XNAME}" --format json | jq '.[] |."params"' | sed "/metal.server/ s|${METAL_SERVER}|${NEW_METAL_SERVER}|" | sed "s/metal.no-wipe=1/metal.no-wipe=0/" | tr -d \");echo $PARAMS;echo --kernel "s3://${S3_ARTIFACT_PATH}/kernel" --initrd "s3://${S3_ARTIFACT_PATH}/initrd";cray bss bootparameters update --hosts "${XNAME}" --kernel "s3://${S3_ARTIFACT_PATH}/kernel" --initrd "s3://${S3_ARTIFACT_PATH}/initrd" --params "${PARAMS}";done
```

Reply:
Agreed that these are good suggestions. They will be addressed with [CASMINST-5574](https://jira-pro.its.hpecorp.net:8443/browse/CASMINST-5574)